artifactory/utils/buildinfoproperties.go

```diff
@@ -202,6 +204,11 @@ func CreateBuildInfoPropertiesFile(buildName, buildNumber string, config *viper.
 		return "", err
 	}
 
+	err = SaveBuildTimestamp(buildName, buildNumber, config)
```

Comment:
Since the *SaveBuildTimestamp* function is used only from this file, it may be better to move it into this file and male it private.
Also, to make the code more consistent, let's call it right after the calls to the *setServerDetailsToConfig* and *setServerDetailsToConfig* functions, and rename it to *setBuildTimestampToConfig*.

Reply:
Done.